components/expenses/ExpenseSummary.js

```diff
@@ -68,6 +68,7 @@ const ExpenseSummary = ({
   const createdByAccount = expense?.requestedByAccount || expense?.createdByAccount || {};
   const expenseItems = expense?.items.length > 0 ? expense.items : expense?.draft?.items || [];
 
+  console.log('exp', expense);
```

Comment:
```suggestion
```

Reply:
That should have been caught by ESLint. `no-console` rule isn't enabled?

Reply:
Yes it's caught, but we only have it as a warning, not a hard error I think; 

https://github.com/opencollective/opencollective-frontend/blob/5ccba0e63b7d1bc904f07c81bdf8c73a5ab16360/.eslintrc.json#L7

The reason is we are using `console.log` and `console.warn` to print out some errors and stuff on the front-end.

Reply:
Oh! it's set to `warn` 😄